site/content/docs/user/quick-start.md

```diff
@@ -151,6 +151,13 @@ kind load docker-image my-custom-image:unique-tag
 kubectl apply -f my-manifest-using-my-image:unique-tag
 ```
 
+**Note**: You can get a list of images present on a cluster node by
+using `docker exec`:
+```
+docker exec -it my-node-name crictl images
```

Comment:
This exposes some of the implementation details of kind which _could_ change in future. The only way to really avoid that though would be to add a CLI command to kind itself for this, which may or may not be desirable.

Overall I think this is meaningful/helpful to include, but maybe needs a warning on it or needs to be made less prominent?

/cc @BenTheElder 
/ok-to-test

Reply:
I also opened #1393 to propose a change to the kind CLI to achieve the same thing.

Reply:
I'm in favor of doing this, I think the future is CRI, and node names are meant to be reliable for this.
if for some reason we support dockershim in the future AND it's baked into kubelet / not CRI compatible still we can just update the note here to show both ways or something